pkg/pb/nodestats.proto

```diff
@@ -36,8 +36,8 @@ message DailyStorageUsageRequest {
 
 message DailyStorageUsageResponse {
     message StorageUsage {
-        double at_rest_total = 2;
-        google.protobuf.Timestamp time_stamp = 3 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+        double at_rest_total = 1;
+        google.protobuf.Timestamp timestamp = 2 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
```

Comment:
Was the previous version pb deployed to production or is this safe to do?

Reply:
It is in master, but that service is not used there. So I am not sure

Reply:
Then we need to test this to ensure that we can deploy this without crashing things.